.eslintrc.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
-  extends: "@mate-academy/eslint-config-react-typescript",
+  extends: "@mate-academy/eslint-config-react",
   // extends: "@mate-academy/eslint-config-react",
```

Comment:
?

Reply:
https://prnt.sc/unjmd5
This was in the description of the task

Reply:
![Screenshot_8](https://user-images.githubusercontent.com/57679291/94239685-d5d59b80-ff1a-11ea-86ec-c56b7c72d117.png)
This was in the description of the task

Reply:
I meant a comment)